src/Arcus.Security.Providers.KeyPerFile/SecretStoreBuilderExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.Extensions.Configuration.KeyPerFile;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Hosting;
+
+namespace Arcus.Security.Providers.KeyPerFile
```

Comment:
```suggestion
// ReSharper disable once CheckNamespace
namespace Microsoft.Extensions.Hosting
```

We tend to use the namespace of the targetted type as the namespace for the extension.

Reply:
Makes sense.  Done.  (Couldn't do this via the suggestion since the files have changed + also had impact on the used namespaces).